install/install_requirements.sh

```diff
@@ -67,6 +67,9 @@ TUNE_NIGHTLY_VERSION=dev20240928
 if [[ -x "$(command -v nvidia-smi)" ]];
 then
   TORCH_NIGHTLY_URL="https://download.pytorch.org/whl/nightly/cu121"
+elif [[ -x "$(command -v rocminfo)" ]];
+then
+  TORCH_NIGHTLY_URL="https://download.pytorch.org/whl/rocm6.1"
```

Comment:
```suggestion
  TORCH_NIGHTLY_URL="https://download.pytorch.org/whl/nightly/rocm6.2"
```
Is there a reason why you're using the rocm stable URL and not the nightly rocm URL?

Reply:
Just a typo - added suggestion to the PR.